### ISSUE TITLE
Rebuilding the PyLadies website & blog

### ISSUE THREAD

**Issue by lorenanicole** [pyladies/project-communications#1]
# Overview

The PyLadies blog has long been inactive on pyladies.com. We should develop a strategy for how to: brand the blog, recruit writers, and develop an outreach strategy for soliciting content. Below are a few questions to help kick off discussion.

## Questions for Discussion

- What should the mission of the PyLadies blog be?
- How does someone submit an article to the blog?
- What does outreach look like for the blog?
- Who is interested in helping work with the proposed PyLadies Project Infra team?
- Anything else

## Issues

From the above we have started spinning out issues on the proirity of work to be done:

**Highest Priority**
- Do research for project team on other blogs: #3 (@Marie-L ) collects list of questions to be answered

**Comment by Marie-L** [comment]
@lorenanicole I'm interested in helping out with this. I can start off by researching other similar blogs, find out what works etc. and then create a projects board to start planning how to tackle this. What do you think ?

**Comment by lorenanicole** [comment]
I think it would be great to put some vision together for sure! But, I would suggest keeping the research focused to tangible action items. Want to start an issue @Marie-L that outlines your objectives for any research?

**Comment by Marie-L** [comment]
@lorenanicole yes I definitely agree ! Ok, I'll do a bit of brainstorming and then will raise a new issue. Thanks!

**Comment by Marie-L** [comment]
hey @lorenanicole [here ](https://github.com/pyladies/project-communications/issues/3) it is

**Comment by lorenanicole** [comment]
@Marie-L btw you can use `#` and then start typing the title of your issue to directly link to it!

**Comment by Marie-L** [comment]
@lorenanicole sure! Thanks for the tip !

**Comment by albendz** [comment]
Created https://github.com/pyladies/pyladies/issues/475 in the infrastructure project to look into this
